evanxd.md

```diff
@@ -1,18 +1,41 @@
-## 07/04 - 07/08 ##
+## 07/11 - 07/15 ##
 
 ### Last week
-* [SensorWeb]
-  - Bug 1284744 - Hide name in the `api.sensorweb.io/users/${userId}` REST API
-  - Bug 1284748 - Survey for finding a good test framework to test the REST APIs
 * [Firefox]
-  - Tried to build Firefox, but got problems. (Will continue to fix it this week.)
+  - Setup development environment
+    - Fixed the try problems: cannot push try
+      - Bug 1286200 - Request hg access be restored
+    - Made a new repo[1] to share documents and config files for new guys
+  - Watch Location Bar module
+    - Bug 647162 - Location Bar does not update
+      - RESOLVED WORKSFORME
+  - Dev Tools
+    - Bug 1282791 - Reps: uninteresting props algorithm is wrong
+      - Fixed the uninteresting props issue
+      - Fixed the max number issue
+    - Rep Tester[2]
+      - Be a collaborator of Rep Tester
```

Comment:
Is this part of DevTools Track 2? What do you expect to get out of this project?